ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-eus-upgrade-from-4.14.yaml

```diff
@@ -178,6 +190,34 @@ tests:
     - ref: hypershift-mce-upgrade
     - chain: openshift-upgrade-qe-test-hypershift
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-hypershift-mce-guest
+- as: power-hypershift-mce-replace-f28
+  cron: 41 6 4 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:multi-target
```

Comment:
If the installation and upgrade both use multi payload, perhaps we can put the job in multi config files ?

Reply:
@liangxia This is EUS upgrade , if want to use the multi config file , I need to create new file for multi, if you think this is the better choice , I could do .

Reply:
Yap, I prefer that way.